prometheusbp/metrics_buckets_spec.go

```diff
@@ -0,0 +1,31 @@
+//go:build newprombuckets
+// +build newprombuckets
+
+package prometheusbp
+
+var (
+	// DefaultLatencyBuckets is the default bucket values for a prometheus histogram metric.
```

Comment:
```suggestion
	// DefaultLatencyBuckets is the default bucket values for a prometheus latency histogram metric.
```

same for the other file.